### DISCUSSION TITLE
TShock release for Terraria 1.4?

### DISCUSSION THREAD

**Discussion by kubedzero** [Pryaxis/TShock#1710] [Q&A] [answered]
Hi all, wanted to start a discussion here regarding Terraria 1.4 "Journey's End" which will be released in a couple weeks on May 16, 2020. 

Will clients and worlds using 1.4 be unable to connect/load into TShock? Furthermore, I wanted to see if TShock has any planned updates that will support the features yet to be announced for 1.4, or if active development has halted.

Finally, I want to see what I can do or what the community can do to make sure TShock remains compatible with this last major version of Terraria (and any minor bugfix versions released soon after). I'd love to see TShock remain relevant (even if the forums go away) with the last version of Terraria that will likely bring tons of new players into the game. 

Thanks for everything so far!

**Comment by kubedzero** [comment, marked as answer]
Closing in favor of active discussion https://github.com/Pryaxis/TShock/discussions

**Reply by hakusaro** [reply]
I believe this is a better link, for the future of searches: https://github.com/Pryaxis/TShock/discussions/1704.

Also, I just wanted to let you k now, @kubedzero: we moved this to a discussion instead of an issue.
